app/controllers/organisations_api_controller.rb

```diff
@@ -53,22 +53,30 @@ def presented_organisation(slug:)
     ).present
   end
 
-  def get_organisations(count:, start:)
-    Services.rummager.search(
-      filter_format: "organisation",
-      order: "title",
-      count: count,
-      start: start
-    )
+  def organisations_from_search(count:, start:)
+    cache_key = "api/organisations/#{count}/#{start}"
+
+    @organisations_from_search ||= Rails.cache.fetch(cache_key, expires_in: 10.minutes) do
```

Comment:
Rather than adding caching everywhere which calls `Services.rummager.search`, could the caching be done there?  Or maybe a new `cached_search` function.  It could take the cache duration as an optional argument, defaulting to 5 minutes, and another optional argument to disable caching if needed.

This works for now, but if any new searches are added people will need to remember to add the explicit caching.

Reply:
Yeah, I was doing this as a short term thing - but might do the rework now as you've moved collections over to search-api already 👍